extensions/blocks/podcast-player/style.scss

```diff
@@ -154,24 +148,28 @@ $block-border-color: $dark-gray-100;
 		color: $text-color-hover;
 		font-weight: bold;
 	}
+}
 
-	.jetpack-podcast-player__episode-link {
-		display: flex;
-		flex-flow: row nowrap;
-		justify-content: space-between;
-		padding: $episode-h-padding $episode-v-padding;
-		text-decoration: none;
-		color: inherit;
-		transition: none;
+// We need to scope this class to override editor link styles.
+.jetpack-podcast-player__episode .jetpack-podcast-player__episode-link {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	padding: $episode-h-padding $episode-v-padding;
+	text-decoration: none;
+	color: inherit;
+	transition: none;
 
-		.jetpack-podcast-player__episode.is-active & {
-			.is-error & {
-				padding-bottom: 0; // Make space for the error element that will be appended.
-			}
-		}
+	&:hover,
+	&:focus {
+		color: $text-color-hover;
 	}
 }
```

Comment:
Why not just nest this within the first declaration of `.jetpack-podcast-player__episode`?

Reply:
Ooooh - the & lifts it out. Nevermind! Got my dev environment working.